src/components/video card/VideoCard.jsx

```diff
@@ -75,6 +98,24 @@ const VideoCard = ({ video, setShowModal, setPlaylistVideo, playlistId }) => {
                   <span className="text-white">Remove from playlist</span>
                 </li>
               )}
+              {!isLiked(video._id) && (
+                <li
+                  className={classes["control-item"]}
+                  onClick={addToLikeHandler}
+                >
+                  <i className="far fa-thumbs-up text-white"></i>
+                  <span className="text-white">Like Video</span>
+                </li>
+              )}
+              {isLiked(video._id) && (
+                <li
+                  className={classes["control-item"]}
+                  onClick={removeLikeHandler}
+                >
+                  <i className="fas fa-thumbs-down text-white"></i>
+                  <span className="text-white">Unlike Video</span>
+                </li>
+              )}
```

Comment:
These both can be combined into 
`{!isLiked(video._id) ? (
                <li
                  className={classes["control-item"]}
                  onClick={addToLikeHandler}
                >
                  <i className="far fa-thumbs-up text-white"></i>
                  <span className="text-white">Like Video</span>
                </li>
              ) : 
               (
                <li
                  className={classes["control-item"]}
                  onClick={removeLikeHandler}
                >
                  <i className="fas fa-thumbs-down text-white"></i>
                  <span className="text-white">Unlike Video</span>
                </li>
              )}`